src/utils/rest-api.ts

```diff
@@ -220,12 +220,19 @@ export function deleteElement(elementUuid: string) {
     });
 }
 
-export function deleteElements(elementUuids: string[], activeDirectory: string) {
+export async function deleteElements(elementUuids: string[], activeDirectory: string) {
     console.info('Deleting elements : %s', elementUuids);
-    const idsParams = getRequestParamFromList('ids', elementUuids).toString();
-    return backendFetch(PREFIX_EXPLORE_SERVER_QUERIES + `/v1/explore/elements/` + activeDirectory + '?' + idsParams, {
-        method: 'delete',
-    });
+    const chunkSize = 50;
+    for (let i = 0; i < elementUuids.length; i += chunkSize) {
+        const partitionIds = elementUuids.slice(i, i + chunkSize);
+        const idsParams = getRequestParamFromList('ids', partitionIds).toString();
+        await backendFetch(
```

Comment:
why do you wait here, why not sending delete requests in parallel ?

Reply:
The idea is usually there will be only one request and sometimes there will be more.

Reply:
and to prevent to send too many requests at the same time

Reply:
I understand the point but I think this is a very important decision (how the big request are cut into parts, the size of the part, if they are simultaneous or not). I Guess it will be applied in many places ?

Do the techlead all agree on this ?

Reply:
I think it's good to split requests in chunks when everything is independant. I don't think we need to overthink it, the chunk size and the parallelism is dependent on the context. Here sequential and chunksize 50 are the good values I think. we may also add a try/catch in the future so that errors in one batch don't affect the subsequent requests